test/js/samples/svelte-element-event-handlers/expected.js

```diff
@@ -39,16 +39,16 @@ function create_dynamic_element(ctx) {
 
 	return {
 		c() {
-			svelte_element1 = element("a");
-			svelte_element0 = element("span");
+			svelte_element1 = element(a);
+			svelte_element0 = element(span);
```

Comment:
After merging https://github.com/sveltejs/svelte/pull/7938, 
`<svelte:element this="p">` is optimized to `<p>`.
Therefore this test failed and I fixed it.